app/src/main/java/org/wikipedia/editactionfeed/provider/MissingDescriptionProvider.kt

```diff
@@ -49,39 +75,57 @@ object MissingDescriptionProvider {
     @Deprecated("Remove when the new API is deployed to production.")
     fun getNextArticleWithMissingDescription(sourceWiki: WikiSite, targetLang: String, sourceLangMustExist: Boolean): Observable<Pair<PageTitle, RbPageSummary>> {
         val targetWiki = WikiSite.forLanguageCode(targetLang)
+        var cachedPair: Pair<PageTitle, PageTitle>? = null
```

Comment:
Should the `cachedPair` be the type of `Pair<PageTitle, RbPageSummary>`?

Reply:
Similarly to the above, we're caching the seed titles that will be used in the requests to get `RbPageSummary`. In this case the caching is even more important because we're caching the results of two different requests.